javascript/javascript1/week1/dog_age.js

```diff
@@ -0,0 +1,14 @@
+const dogYearOfBirth = 1957;
+const dogYearFuture = 2027;
+console.log(dogYearFuture - dogYearOfBirth);
+const dogYear = 60;
+const dog_age= dogYearFuture - dogYearOfBirth;
```

Comment:
Remember to always use [camel case](https://en.wikipedia.org/wiki/Camel_case) in Javascript.